src/utils/s3_mods.py

```diff
@@ -0,0 +1,148 @@
+# General functions
+import json
+import pandas as pd
+from io import StringIO
+import logging
+from typing import List
+import subprocess
+import os
+
+# Functions specific to s3 bucket
+import boto3
+import raz_client
+from rdsa_utils.cdp.helpers.s3_utils import file_exists, create_folder_on_s3
+
+
+# set up logging
+s3_logger = logging.getLogger(__name__)
+
+
+def create_client(config):
+    client = boto3.client("s3")
+    raz_client.configure_ranger_raz(client, ssl_file=config["s3"]["ssl_file"])
+    return client
```

Comment:
docstring + type hinting

Reply:
Created a docstring and a type hint for config. Return type is a boto3 client. It's not so easy, and I only found this resource: https://pypi.org/project/boto3-stubs/. I think it's best not to overload the pipeline with dependencies. 

Any thoughts how to do it better are most welcome

Reply:
I agree that providing no return type is the correct implementation here. resolving this conversation.